internal/gatewayapi/route.go

```diff
@@ -1313,6 +1314,7 @@ func (t *Translator) processDestination(backendRefContext BackendRefContext,
 	}
 
 	ds.Weight = &weight
+	ds.IPFamily = getServiceIPFamily(resources.GetService(backendNamespace, string(backendRef.Name)))
```

Comment:
better to move to L1282, only service support IPFamily now.

Reply:
Thanks for the feedback! I’ve updated the code.